src/Optimization/UI/AxisymmetricOptimizationUI.html

```diff
@@ -0,0 +1,223 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+<meta content="en-us" http-equiv="Content-Language" />
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+
+<style>
+div.fixed {
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  width: 300px;
+  border: 3px solid #73AD21;
+}
+.auto-style5 {
+	text-align: center;
+	border: 1px solid #00FF00;
+}
+.auto-style6 {
+	text-align: center;
+}
+.auto-style7 {
+	text-align: center;
+	font-size: xx-large;
+}
+
+</style>
```

Comment:
We should split this off (and all the other coloring code which is done elsewhere in Julia) into a CSS file.